init/common.js

```diff
@@ -189,6 +197,18 @@ const createProject = async (interestId, userId) => {
   });
 };
 
+const createProjectWithoutTeam = async (interestId, userId) => {
+  const service = strapi.service("api::project.project");
+  let existing = await service.findOne(interestId);
```

Comment:
This looks a bit weird that we are trying to find project by interest ID?

Reply:
Thank you pointing out. Updated.